docs/api/options.md

```diff
@@ -74,6 +74,37 @@ const wrapper = shallowMount(Component, {
 expect(wrapper.find('div')).toBe(true)
 ```
 
+You can also provide slots components with props.
```

Comment:
I like this a lot, but can you add the context for _why_ this is useful? If you used your Pull Request description (including component code snippets) that would be perfect. It didn't make any sense until I saw what the template for "Component" looked like.

Reply:
I'm working on it and trying to merge two snippets into one. The original snippet is not including the stubs for `<my-component />` so should I remove mine also?

Reply:
Line 70: 
```
- qux: '<my-component />'
+ qux: '<my-component foo="bar" :foo="val"/>'
```
and `mocks` for `val`?

Reply:
Ah! I see... I missed the original snippet above 😄If you'd like to merge and refine yours more, please feel free to. I do like the full context that your PR example has and think it would help illustrate the "Why" a bit more